sleap_nn/inference/predictors.py

```diff
@@ -1242,6 +710,7 @@ class (doesn't return a pipeline) and the Thread is started in
 
         elif self.provider == "VideoReader":
             provider = VideoReader
+            self.preprocess = True
```

Comment:
Instance attributes MUST be defined at the top of the class! This means either as an `attrs.field` in the body of the class, or in the constructor.